src/js/views/eva-variant-view.js

```diff
@@ -674,7 +759,7 @@ EvaVariantView.prototype = {
             return '<tr>' + rowContent + '</tr>';
         };
 
-        var summaryDisplayFields = {organism : "Organism", assembly: "Assembly", submitterHandle: "Study", chromosome: "Contig", start: "Start",
+        var summaryDisplayFields = {organism : "Organism", assembly: "Assembly", submitterHandle: "Study", contig: "Contig/Chromosome accession", chromosome: "Chromosome", start: "Start",
```

Comment:
consider putting "Chromosome" first. the accession column title catches the sight first, and then I moved to the rows, without noticing that there's a chromosome column title.

Reply:
Someone comparing multiple SS in the variant view against the release VCF will find it easier to have chr + start lined up.